### PR TITLE
Fix incorrect 128bit uuids and add verification

### PR DESCRIPTION
**Pull request by bencefr**

### CODE REVIEW

File: verify.js [hunk 1]
```diff
@@ -35,9 +35,18 @@ const noDuplicates = Object.keys(db)
         console.error(`Failed to verify '${list}': '${numberOfItems - numberOfUniqueIdentifiers}' duplicate Identifier(s) found.`);
         return false;
       }
+
+      const uuidtest = db[list].map(({ uuid }, i) => {
+        if (! /^([A-F0-9]{4}|[A-F0-9]{8}-[A-F0-9]{4}-[A-F0-9]{4}-[A-F0-9]{4}-[A-F0-9]{12})$/.test(uuid)) {
```
Comment (philips77) [inline verify.js:40]: 32-bit UUIDs are also allowed by spec, although not defined yet.

File: verify.js [hunk 2]
```diff
@@ -35,9 +35,18 @@ const noDuplicates = Object.keys(db)
...
+          console.error(`Found incorrect uuid ${uuid} of item #${i} in ${list}`);
+          return false;
+        }
+        return true;
+      }).every(v => v);
+      if (! uuidtest) return false;
```
Comment (dinesharjani) [inline verify.js:46]: Looks Good

File: verify.js (hunk 1 above)
Comment (dinesharjani) [reply verify.js:40]: @bencefr we can add them in this change or in a different one; either way is fine by me

Comment (bencefr) [reply verify.js:40]: @bihanssen is looking at `ajv` if it is possible to validate the field by the schema. Maybe we'll move and extend the validation there.

Comment (bihanssen) [reply verify.js:40]: I'm ok with either solution. I haven't started on any implementation, just confirmed from the doc that ajv supports direct checking. I'll leave it to you to decide where to put it.

Comment (dinesharjani) [reply verify.js:40]: I think we should merge this, and then continue in a separate change.

**Pull request merged**